src/config.ts

```diff
@@ -2,6 +2,7 @@ import OAuth2Client, { OAuth2ClientConstructor } from "@fwl/oauth2";
 
 type Config = {
   connectDomain: string;
+  connectTheme: string;
```

Comment:
shouldn't it be optional?

Reply:
I think leaving this as it is, but changing the field's value attribution like this should be better:
```ts
config.connectTheme = process.env.CONNECT_ACCOUNT_THEME || "fewlines";
```
What do you think?

Reply:
👍